### PR TITLE
CTP-3652 support H5P marks transfer

### PR DESCRIPTION
**Pull request by watson8**
Draft pull request for review as discussed. I did not yet increment the version number as I have a separate branch for each and I do not know which is likely to be merged first so thought it best to avoid conflict. There are similar branches for the other 3 mod types (Lesson, LTI, HVP) under https://github.com/ucl-isd/moodle-local_sitsgradepush/branches

### CODE REVIEW

File: classes/submission/hvp.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace local_sitsgradepush\submission;
+
+/**
+ * Class for coursework plugin (mod_coursework) submission.
```
Comment (aydevworks) [inline classes/submission/hvp.php:20]: Class for hvp?

**Review by aydevworks — COMMENTED** [review]
Done review.

File: classes/submission/hvp.php (hunk 1 above)
Comment (watson8) [reply classes/submission/hvp.php]: Changed in commit 4c30c60

**Comment by aspark21** [comment]
Superseeded by #71

**Pull request closed without merging**